### PR TITLE
Update fix for MW65_rgb and added personal keymap

### PR DESCRIPTION
**Pull request by HorrorTroll**
## Description
- Update fix for using 6 rows instead 5 rows, pin B0 was unused
- No need RGBLIGHT, since this one wasn't used
- Correct LED flag for underglow, key light, modifier, indicator
- Re-calculation LED position
- ~~Giving correct layout name to 65_ansi_blocker~~
- Added 2 personal keymap for me and my friend (he doesn't know coding, sorry)

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [x] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/mwstudio/mw65_rgb/config.h [hunk 1]
```diff
@@ -43,19 +43,11 @@
...
-
-    #define RGBLIGHT_HUE_STEP 8
-    #define RGBLIGHT_SAT_STEP 8
-    #define RGBLIGHT_VAL_STEP 8
-#endif
+
+#define RGB_MATRIX_MAXIMUM_BRIGHTNESS 200
+#define RGB_MATRIX_STARTUP_VAL RGB_MATRIX_MAXIMUM_BRIGHTNESS
+#define DRIVER_LED_TOTAL 83
+#define RGB_MATRIX_KEYPRESSES
```
Comment (drashna) [inline keyboards/mwstudio/mw65_rgb/config.h:50]: just a heads up, for RGB Matrix, you now need to explicitly enable any modes you want to to be used.

Comment (HorrorTroll) [reply keyboards/mwstudio/mw65_rgb/config.h]: Just resolved issue, thanks

File: keyboards/mwstudio/mw65_rgb/keymaps/horrortroll/config.h [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+/* Forcing to use NKRO instead 6KRO */
+#define FORCE_NKRO
+
+/* Change USB Polling Rate to 1000hz and a larger keys per scan for elite gaming */
+#define USB_POLLING_INTERVAL_MS 1
+#define QMK_KEYS_PER_SCAN 12
+
+/* RGB Matrix config */
+#define DISABLE_RGB_MATRIX_BAND_SAT
```
[the remark this answers is not in the document]
Comment (HorrorTroll) [reply keyboards/mwstudio/mw65_rgb/keymaps/horrortroll/config.h]: Just resolved issue, thanks

**Review by drashna — APPROVED** [review]

**Review by tzarc — CHANGES_REQUESTED** [review]
C/H Source files all need to have appropriate license headers, as per the [PR Checklist](https://docs.qmk.fm/#/pr_checklist?id=example-gplv2-header).

Might be wise giving the whole checklist a look through to see if there are any others that need fixing up, too.

**Comment by HorrorTroll** [comment]
Thanks for suggest changes, done for fixed them

File: keyboards/mwstudio/mw65_rgb/rules.mk [hunk 3]
```diff
@@ -16,6 +16,11 @@ NKRO_ENABLE = yes           # Enable N-Key Rollover
 BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
 RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
 AUDIO_ENABLE = no           # Audio output
+
 RGB_MATRIX_ENABLE = yes
 RGB_MATRIX_DRIVER = WS2812
+
 ENCODER_ENABLE = yes
+
+LAYOUTS = 65_ansi_blocker
```
Comment (noroadsleft) [inline keyboards/mwstudio/mw65_rgb/rules.mk:25]: This won't actually work – [the `65_ansi_blocker` community layout has 67 keys](https://github.com/qmk/qmk_firmware/blob/7c186ebb974b4bbadbb74a7610bab5d2f66d030e/layouts/default/65_ansi_blocker/default_65_ansi_blocker/keymap.c#L17-L23), but the layout macro has 69, due to the assignments for the rotation of the rotary encoder.

I'd prefer the layout macro name remain as `LAYOUT` as well.

```suggestion
```

**Review by noroadsleft — CHANGES_REQUESTED** [review]

File: keyboards/mwstudio/mw65_rgb/rules.mk (hunk 3 above)
Comment (HorrorTroll) [reply keyboards/mwstudio/mw65_rgb/rules.mk]: Done, thanks

**Review by noroadsleft — APPROVED** [review]
Looks good to me.

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**